src/Modules/Shared/Microsoft.PowerShell.Archive/Microsoft.PowerShell.Archive.psd1

```diff
@@ -2,7 +2,7 @@
 GUID="eb74e8da-9ae2-482a-a648-e96550fb8733"
 Author="Microsoft Corporation"
 CompanyName="Microsoft Corporation"
-Copyright="� Microsoft Corporation. All rights reserved."
+Copyright="© Microsoft Corporation. All rights reserved."

```

Comment:
Some Windows tools will not handle this file correctly because there is no BOM.  For example, gvim shows 2 characters for the copyright symbol, I assume because it guesses the file is Latin-1, not UTF8.